components/tooltip/stories/tooltip.stories.js

```diff
@@ -95,6 +96,53 @@ export default {
 export const Default = PlacementVariants.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+/**
+ * A tooltip is positioned in relation to its target. Placement property values are at the: top, top left, top right,
+ * top start, top end, bottom, bottom left, bottom right, bottom start, bottom end, left, left top, left bottom, start,
+ * start top, start bottom, right, right top, right bottom, end, end top, end bottom. The default placement value is at
+ * the top.
+ *
+ * Placement property values containing the words "start" and "end" behave like logical properties and will change side
+ * if text direction is changed.
+ */
+export const Placement = TooltipPlacementGroup.bind({});
+Placement.tags = ["!dev"];
+Placement.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A tooltip that shows on hover using CSS only. Note that this approach does not support text wrapping. Also, note
+ * that these Tooltips will likely not work on touch-enabled devices without additional client-side scripting.
+ */
+export const ShowOnHover = TooltipShowOnHover.bind({});
```

Comment:
This was a great callout. I honestly didn't even notice that `showOnHover` arg in the Template until you mentioned it. Taking into consideration the other comments about `ShowOnHover`, here's the conclusion I've come to on this:
I played around with it for a little while and I think the main issue with the `showOnHover` logic in the template is this (as you already pointed out): if it gets turned on for one story, anywhere in Tooltip (in the sidebar, for instance), it gets turned on EVERYWHERE, which makes the other tooltips look weird. That's because we're not cleaning up and removing the class for stories that don't use `showOnHover`. I added that bit in to clean up and remove the class if it's not being used, but then I realized that doing it that way doesn't work on the Docs page (where we show) as far as I can think of.

So, I think the options are these:
* Have a separate story for "Show on Hover" in the sidebar, and remove it from Storybook. I believe that might be how I have it currently. If we do this, we can use the showOnHover control as it was intended to be used, but then it's not displayed or described on the Docs page (that JSDoc comment goes nowhere, basically).
* Keep it as it was before on the Docs page (revert the last commit). It doesn't use the showOnHover control, that class is hardcoded into the story instead (because of how the showOnHover control takes over if it's turned on at all).
    * If we do this, I'm wondering when we would ever use that showOnHover control, and it might be wise to consider its removal? I'm more inclined to make that a different card, because that seems like a bigger decision and I'd maybe want to consider the history of why it was added and if there are any other unintended side effects before taking it out.

I think I might be more partial to the second option because we wouldn't really need to use any other controls in conjunction with show on hover (especially once all the other variants aside from neutral go away), but on the other hand, you don't necessarily expect to be able to play with the component on the Docs page like you would here. Your thoughts?

Reply:
Well thanks for looking into this! Yeah I think I agree with you. I vote to revert the last commit, without using that `showOnHover` control. Like you said, it sort of feels like a bigger decision or at least a bigger discussion on what that control is supposed to do, or if we need it at all.